python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -5688,3 +5688,153 @@ def test_dataframe_assign_scalar_with_scalar_cols(col_data, assign_val):
     gdf["b"] = assign_val
 
     assert_eq(pdf, gdf)
+
+
+# flake8: noqa : W291
+def test_dataframe_info():
+    import io
+
+    buffer = io.StringIO()
+    str_cmp = """<class 'cudf.core.dataframe.DataFrame'>
```

Comment:
Rather than one long multiline string, we could use several concatenated strings. This will help keep things nicely indented. Alternatively, we could use [`textwrap.dedent`](https://docs.python.org/3.8/library/textwrap.html#textwrap.dedent) with a multi-line string.

Reply:
Used `.dedent` with multi-line strings..

Thanks @shwina, TIL `textwrap.dedent`.